.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,6 @@
+Fixes issue # .
```

Comment:
I think GitHub special keyword is `Fixes #{issue_number}` for automatic closing of issue. 🤔

Reply:
If you can find it in their docs, that would be great!  See the links I provided above.

Reply:
https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue
See the linking sub-heading in the above doc!

Reply:
It's a template, so as long as we have a reminder to them to type in the issue number.  The autosuggest of Issues appears after someone types the number sign # only in fact.   But sure, we can give them a larger hint reminder like you have suggested if you think is needed.

Reply:
No I was talking about the change `Fixes issue #` to `Fixes #` writing issue ahead of the # will not close the issue.

Reply:
Ah the "supported keywords"!  Yes you are right.